app/assets/stylesheets/pages/_votes_new.scss

```diff
@@ -144,8 +141,14 @@ html, body {
     margin: 10px;
   }
 
+  #ninja-submit, #ninja-submit2{
+    position: sticky;
+    bottom: 0;
+  }
+
   #nav-tabContent {
     max-width: 100vw;
-    max-height: 700px;
+    max-height: 570px;
```

Comment:
this is dope, glad we did this! one thought - might be better to set the max height based on a percentage (using vh) so it renders slightly differently but proportionately on different screens?

Reply:
I played around with this one for a while before I landed on that height. It won't really affect the screen you're working on and limiting it to that height helps avoid the disappearing check mark. you're welcome to change it to vh if you want though. I need a break from that page